.travis.yml

```
@@ -0,0 +1,11 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+jdk:
+  - oraclejdk8
+script: rake test:install-core && rake test:core-fail-fast
```

We want to fail fast, I would prefer to see a complete run?